README.md

```diff
@@ -14,20 +14,27 @@ You cannot run the image directly because it does not come with the Pega web app
 
 The simplest way to use this image is to create your own Dockerfile with contents similar to the example below and extract the Pega distribution to the same directory as the Dockerfile.  It's a best practice to build this image on a Linux system to retain proper file permissions.  Replace the source paths with the actual paths to the Pega Infinity software libraries and specify a valid JDBC driver for your target database.
 
-    FROM pegasystems/pega-ready
+    FROM pegasystems/pega-ready as release
+
+    FROM release as unzipprweb
+    RUN apt-get-update && \
```

Comment:
Should this be added in the _ready_ image?  Or is the intent that this is a builder image? (I think I am just confused by unzipprweb).

Reply:
This is not in _pega-ready_ image DockerFile. This is **README** for end-users on how to support unzip for a secure image and perform unzip of prweb upon mounting.